src/components/AboutHome/AboutHome.js

```diff
@@ -29,6 +30,8 @@ const AboutHome = ({ user, setCurrentLink }) => {
   const [monitorPlanContent, setMonitorPlanContent] = useState();
   const [qaCertificationContent, setQACertificationContent] = useState();
 
+  const [warningMessage, setWarningMessage] = useState(null);
```

Comment:
This state definition is not being used anywhere in a meaningful way (we can remove this). See comment below.